q2_fmt/plugin_setup.py

```diff
@@ -360,4 +368,62 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.peds_simulation,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata, 'time_column': Str,
+                'reference_column': Str, 'subject_column': T_subject,
+                'filter_missing_references': Bool,
+                'drop_incomplete_subjects': Bool,
+                'drop_incomplete_timepoint': List[Str],
+                'replicates': Int % Range(999, None)},
+    outputs=[('per_subject_stats', StatsTable[Pairwise]),
+             ('global_stats', StatsTable[Pairwise])],
+    parameter_descriptions={
+        'metadata': 'The sample metadata.',
+        'time_column': 'The column within the `metadata` that the'
```

Comment:
Nit picky, but since your using the tick marks around *metadata*, I think you should be treating it as a proper noun. This should be updated throughout the other descriptions as well.  

```suggestion
        'time_column': 'The column within `metadata` that the'
```